### PR TITLE
Swap flatbed trucks for transports in 3CB template transport slots

### LINKED ISSUE #1462: Flatbed trucks used in templates where passenger vehicles should be
**Describe the bug**
The vehNATOTrucks, vehCSATTrucks and vehFIATruck variables should only contain passenger vehicles, as these are assumed to be capable of carrying infantry squads. Flatbed trucks can be moved to vehNATO/CSATCargoTrucks.

Known cases: 
UK3CB_TKA_I_V3S_Recovery in 3CB_Occ_TKA_Arid
UK3CB_TKA_B_Ural_Recovery in 3CB_Occ_BAF_Arid

There may be others.

### PR DESCRIPTION
**Pull request by jaj22**
## What type of PR is this.
1. [X] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Some 3CB templates have flatbed trucks (no passenger capability) in transport vehicle template slots. This will cause broken QRFs, attacks and reinforcements. This PR switches them to suitable transport trucks.

### Please specify which Issue this PR Resolves.
closes #1462

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### CODE REVIEW

**Comment by SonorousMeerkat** [comment]
Bear in mind that the MAN trucks cannot load crates.

**Comment by SonorousMeerkat** [comment]
The flatbeds were used to make sure that the outpost had a vehicle that could take the loot crate.

File: A3-Antistasi/Templates/3CB_Occ_BAF_Arctic.sqf [hunk 1]
```diff
@@ -160,7 +160,7 @@ vehNATOAir = vehNATOTransportHelis + vehNATOAttackHelis + [vehNATOPlane,vehNATOP
 if ((gameMode != 4) and (!hasFFAA)) then
 	{
 	vehFIAArmedCar = "UK3CB_BAF_LandRover_WMIK_GPMG_FFR_Green_B_DPMT";
-	vehFIATruck = "UK3CB_BAF_MAN_HX60_Cargo_Green_A_Arctic";
+	vehFIATruck = "UK3CB_BAF_MAN_HX60_Transport_Green_DPMT";
```
Comment (Bob-Murphy) [inline A3-Antistasi/Templates/3CB_Occ_BAF_Arctic.sqf:163]: Please revert back to flatbed.

File: A3-Antistasi/Templates/3CB_Occ_BAF_Temp.sqf [hunk 2]
```diff
@@ -160,7 +160,7 @@ vehNATOAir = vehNATOTransportHelis + vehNATOAttackHelis + [vehNATOPlane,vehNATOP
 if (gameMode != 4) then
 	{
 	vehFIAArmedCar = "UK3CB_BAF_LandRover_WMIK_GPMG_FFR_Green_B_DPMT";
-	vehFIATruck = "UK3CB_BAF_MAN_HX60_Cargo_Green_A_DPMT";
+	vehFIATruck = "UK3CB_BAF_MAN_HX60_Transport_Green_DPMT";
```
Comment (Bob-Murphy) [inline A3-Antistasi/Templates/3CB_Occ_BAF_Temp.sqf:163]: Please revert back to flatbed.

File: A3-Antistasi/Templates/3CB_Occ_BAF_Trop.sqf [hunk 3]
```diff
@@ -161,7 +161,7 @@ vehNATOAir = vehNATOTransportHelis + vehNATOAttackHelis + [vehNATOPlane,vehNATOP
 if (gameMode != 4) then
 	{
 	vehFIAArmedCar = "UK3CB_BAF_LandRover_WMIK_GPMG_FFR_Green_B_Tropical";
-	vehFIATruck = "UK3CB_BAF_MAN_HX60_Cargo_Green_A_Tropical";
+	vehFIATruck = "UK3CB_BAF_MAN_HX60_Transport_Green_Tropical";
```
Comment (Bob-Murphy) [inline A3-Antistasi/Templates/3CB_Occ_BAF_Trop.sqf:164]: Please revert back to flatbed.

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]
Please adapt.
Meerkat wrote the reason above.

**Review by Bob-Murphy — APPROVED** [review]

**Comment by jaj22** [comment]
@Bob-Murphy @SonorousMeerkat @HakonRydland @Lazejun 

Not fixed because flatbed trucks still don't carry passengers and so shouldn't be used for QRFs and reinforcements. If it's considered important to have a cargo truck in outposts/resources/etc then I could add some filter code in those functions to make sure that it's only placing cargo-capable trucks.

**Pull request merged**